lib/lookerUtils.ts

```diff
@@ -0,0 +1,35 @@
+import { getProposedEndDate } from "./experimentUtils";
+
+// XXX getExperimentLookerDashboardDates and getLookerSubmissionTimestampDateFilter
+// were moved to this file rather than looker.ts due to "Module not found" errors.
+// These errors were likely made because looker.ts has functions that use node
+// modules that should only exist in server components. But these functions are 
+// passed through other functions that eventually go through columns.tsx which has
+// "use client". We should look fo a better architectural solution regarding 
+// the situation with these errors.
+
+/**
+ * @returns the proposed end date used for experiment Looker dashboard links. 
+ *          We are adding 1 to the date because the end date in the Looker
+ *          date filters are not inclusive. 
+ */
+export function getExperimentLookerDashboardDates(startDate : string | null, proposedDuration : number | undefined) {
```

Comment:
Since this doesn't actually return multiple dates, how about renaming this `getLookerDashboardProposedEndDate` for clarity when reading calling sites?

Reply:
That makes sense 👍